lookout/style/typos/cmdline_tools.py

```diff
@@ -0,0 +1,291 @@
+"""
+Command line utilities to check the quality of a model on a given dataset, visualize errors, etc.
+"""
+from argparse import ArgumentParser
+from typing import Any
+
+from lookout.core.cmdline import ArgumentDefaultsHelpFormatterNoNone
+from lookout.core.slogging import setup as setup_slogging
+
+from lookout.style.typos.train_test_corrector import (DEFAULT_FREQUENCY_PATH,
```

Comment:
Better to spread these across the page width - each new imported item adds a whole line otherwise